### DISCUSSION TITLE
Book 3 Listing 39: `plane_origin`, `axis_A`, and `axis_B`

### DISCUSSION THREAD

**Discussion by davidfrogley** [RayTracing/raytracing.github.io#1255] [Debugging]
The code in Listing 39 (`quad.h`) in the override of the `random` method lists three member fields by, I believe, the wrong name (unless I missed something, which is certainly possible):
- `plane_origin` should be `Q`
- `axis_A` should be `u`
- `axis_B` should be `v`

![image](https://github.com/RayTracing/raytracing.github.io/assets/6363895/ac894ccf-3d55-4f27-80a7-5210d689907c)

**Comment by hollasch** [comment]
Spawned issue #1266.
